src/interfaces/TagsListOptions.ts

```diff
@@ -0,0 +1,9 @@
+import { InlcudeRelatedResourcesOptions } from './InlcudeRelatedResourcesOptions';
+import { ListOptions } from './ListOptions';
+
+export interface TagsListOptions
+  extends ListOptions,
+    InlcudeRelatedResourcesOptions {
```

Comment:
you only need to extend this interface if the endpoint you are working with provides some included related resources: see https://www.terraform.io/docs/cloud/api/index.html#inclusion-of-related-resources

For example, the workspace endpoint has related resources: https://www.terraform.io/docs/cloud/api/workspaces.html#available-related-resources
But the tags endpoint doesn't: https://www.terraform.io/docs/cloud/api/organization-tags.html